samples/unit-tests/axis/stacklabels/demo.js

```diff
@@ -21,27 +21,26 @@ QUnit.test('Stack labels on non-data axis', function (assert) {
     });
 
 
-    assert.close(
+    assert.strictEqual(
         chart.container.querySelector('.highcharts-stack-labels text')
-            .getBBox().y,
-        0,
-        1,
-        'The y attribute is around to 0.'
+            .getAttribute('y') !== null,
+        true,
```

Comment:
Instead of `strictEqual` with an !== expression in the first parameter, use `notStrictEqual`: